packages/one-app-dev-bundler/utils/bundle-external-fallbacks.js

```diff
@@ -69,6 +71,27 @@ export const bundleExternalFallbacks = async () => {
         console.error(`Failed to build fallback for external ${externalName} for ${env}`, error);
       }
     }))));
+
+    const integrityManifest = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'bundle.integrity.manifest.json'), 'utf-8'));
+    const requiredExternalsData = requiredExternals.reduce((obj, externalName) => {
+      const version = readPackageUpSync({
+        cwd: path.resolve(process.cwd(), 'node_modules', externalName),
+      })?.packageJson.version;
+      const semanticRange = packageJson.dependencies[externalName];
+      return {
+        ...obj,
+        [externalName]: {
+          name: externalName,
+          version,
+          semanticRange,
+          integrity: integrityManifest[`${externalName}.browser`], // prevent breaking change
+          browserIntegrity: integrityManifest[`${externalName}.browser`],
```

Comment:
I am not 100% on if we need `browserIntegrity` and `integrity`

Reply:
is `browserIntegrity` used anywhere already, or is that new to this PR? I don't think it makes sense to maintain 2 of the same values, even though it is a bit more clear

Reply:
its not used anywhere, but integrity is. Id be more inclined to remove integrity, especially as i don't think production builds currently have integrity included